### PR TITLE
javac, curl and mill

### PR DESCRIPTION
**Pull request by joan38**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -136,11 +136,7 @@ lazy val `sbt-plugin` = myCrossProject("sbt-plugin")
 
 lazy val `mill-plugin` = myCrossProject("mill-plugin")
   .settings(
-    crossScalaVersions := Seq(Scala213, Scala212),
```
Comment (fthomas) [inline build.sbt:139]: I think `crossScalaVersions` is required so that the mill plugin is published for Scala 2.12 and 2.13.

Comment (joan38) [reply build.sbt:139]: Removed by accident. Fixed now.

File: project/Dependencies.scala [hunk 2]
```diff
@@ -1,6 +1,11 @@
 import sbt._
+import sbt.Keys._
 
 object Dependencies {
+  val mill = Def.setting {
+    val version = if (scalaBinaryVersion.value == "2.12") "0.6.3" else "0.7.4"
```
Comment (lefou) [inline project/Dependencies.scala:6]: Just some food for thought: I think at compile time you should bind to the lowest API denominator, `0.6.0` resp. `0.7.0`. And only use newer versions at runtime. (Can't tell you if and how that is possible with sbt.)

Comment (joan38) [reply project/Dependencies.scala:6]: Good point. We should do that in a separate PR I think.

**Pull request merged**